docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -0,0 +1,130 @@
+
+[role="screenshot"]
+image::images/install-endpoint/ransomware-protection.png[Detail of ransomware protection section.]
+
+[discrete]
+[[memory-protection]]
+== Memory threat protection
+
+Memory threat protection detects and stops in-memory threats on Windows systems, such as shellcode injection, which are used to evade traditional file-based detection techniques.
+
+Memory threat protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, memory threat protection will be disabled by default.
+
+Memory threat protection levels are:
+
+* **Detect**: Detects memory threat activity on the host and generates an alert. The {agent} will **not** block the in-memory activity. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects memory threat activity on the host, forces the process or thread to stop, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {rule}` syntax.
+
+[role="screenshot"]
+image::images/install-endpoint/memory-protection.png[Detail of memory protection section.]
+
+[discrete]
+[[behavior-protection]]
+== Malicious behavior protection
+
+Malicious behavior protection detects and stops threats by monitoring the behavior of system processes for suspicious activity. Behavioral signals are much more difficult for adversaries to evade than traditional file-based detection techniques.
+
+Malicious behavior protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, malicious behavior protection will be disabled by default.
+
+Malicious behavior protection levels are:
+
+* **Detect**: Detects malicious behavior on the host and generates an alert. The {agent} will **not** block the malicious behavior. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects malicious behavior on the host, forces the process to stop, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {rule}` syntax.
+
+[role="screenshot"]
+image::images/install-endpoint/behavior-protection.png[Detail of behavior protection section.]
+
+[discrete]
+[[event-collection]]
+== Event collection
+
+In the **Settings** section, review the events that collect data on each operating system. By default, all event data is collected. If you no longer want a specific event to collect data, deselect it.
```

Comment:
The events don't collect data, right? Maybe we should say "...review the events that the agent will collect on each operating system. By default, all events are collected. If you no longer want to collect a specific type of event, deselect it."

Reply:
Good catch, thanks! Added a slightly tweaked version of your suggestion.